<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_OgeWiNavigationTab</class>
 <widget class="QWidget" name="C_OgeWiNavigationTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_OgeWiNavigationTab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgeGbxNavigationTab" name="pc_GroupBoxStyle">
     <property name="title">
      <string>StyleBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>19</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="stw::opensyde_gui_elements::C_OgeLabSvgOnly" name="pc_LabelIcon">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string>KEEP THE MARGINS!</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelText">
        <property name="text">
         <string>KEEP THE MARGINS!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="pc_FrameBottomLine">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabSvgOnly</class>
   <extends>QLabel</extends>
   <header>C_OgeLabSvgOnly.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGenericNoPaddingNoMargins.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeGbxNavigationTab</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxNavigationTab.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
